imagepruner/v1/00-crd.yaml

```diff
@@ -0,0 +1,742 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
are we not going to have a [clash](https://github.com/openshift/cluster-image-registry-operator/blob/master/Dockerfile#L9) in naming as this CRD and the one under `imageregistry` have the same name: "00-crd.yaml" ?

Reply:
fixed, changed to 01-crd.yaml when moving to imageregistry/v1 (but i don't think it would have clashed with it's previous name/location, but it doesn't matter now)